model/config.go

```diff
@@ -1606,16 +1609,24 @@ func (s *PluginSettings) SetDefaults() {
 
 type MessageExportSettings struct {
 	EnableExport        *bool
+	ExportFormat        *string
```

Comment:
This used to just default to "actiance", but now that we have support for GlobalRelay, the user needs to be able to configure it for use.